### PR TITLE
fix: #845 improve log output 

### LINKED ISSUE #845: improve log output
When the publish pipe runs, some of the publishers descriptions start with `creating `, which makes the output weird.

We should probably drop those `creating ` prefixes.

We should also find a way to make it clearer that some logs are part of one step, eg:

```
   • PUBLISHING ARTIFACTS
      • releasing to s3
      • skipped                   reason=s3 section is not configured
      • releasing with HTTP PUT
      • skipped                   reason=put section is not configured
      • releasing to Artifactory
      • skipped                   reason=artifactory section is not configured
      • releasing to GitHub
      • creating or updating release repo=caarlos0/couchbase-exporter tag=v0.0.2
      • release updated           url=https://github.com/caarlos0/couchbase-exporter/releases/tag/v0.0.2
      • uploading to release      file=dist/couchbase-exporter_0.0.2_checksums.txt name=couchbase-exporter_0.0.2_checksums.txt
      • uploading to release      file=dist/couchbase-exporter_0.0.2_darwin_amd64.tar.gz name=couchbase-exporter_0.0.2_darwin_amd64.tar.gz
      • uploading to release      file=dist/couchbase-exporter_0.0.2_linux_amd64.tar.gz name=couchbase-exporter_0.0.2_linux_amd64.tar.gz
      • creating homebrew formula
      • skipped                   reason=brew section is not configured
      • creating scoop manifest
      • skipped                   reason=scoop section is not configured
      • creating Docker images
      • pushing docker image      image=caarlos0/couchbase-exporter:latest
      • pushing docker image      image=caarlos0/couchbase-exporter:v0.0.2
      • Snapcraft Packages
   • release succeeded after 31.34s
```

its clear that `releasing to s3` and etc are under `publishing artifacts`, but it may not be clear that where on publisher starts or ends and also what logs are part of what publisher... maybe add another indentation level?

### PR DESCRIPTION
**Pull request by julio-garcia-fc**
I want to fix #845 
Removed the "creating" prefixes.

### CODE REVIEW

File: internal/pipe/nfpm/nfpm.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func create(ctx *context.Context, format, arch string, binaries []artifact.Artif
 	}
 
 	var path = filepath.Join(ctx.Config.Dist, name+"."+format)
-	log.WithField("file", path).Info("creating")
+	log.WithField("file", path)
```
Comment (caarlos0) [inline internal/pipe/nfpm/nfpm.go]: this one we can leave as it was before:

```suggestion
	log.WithField("file", path).Info("creating")
```

File: internal/pipe/release/release.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func doPublish(ctx *context.Context, c client.Client) error {
 	}
 	log.WithField("tag", ctx.Git.CurrentTag).
 		WithField("repo", ctx.Config.Release.GitHub.String()).
-		Info("creating or updating release")
+		Info("release")
```
Comment (caarlos0) [inline internal/pipe/release/release.go]: this one too:

```suggestion
		Info("creating or updating release")
```

File: internal/pipe/snapcraft/snapcraft.go [hunk 3]
```diff
@@ -133,7 +133,7 @@ func create(ctx *context.Context, arch string, binaries []artifact.Artifact) err
 	}
 
 	var file = filepath.Join(primeDir, "meta", "snap.yaml")
-	log.WithField("file", file).Debug("creating snap metadata")
+	log.WithField("file", file).Debug("snap metadata")
```
Comment (caarlos0) [inline internal/pipe/snapcraft/snapcraft.go]: here too:

```suggestion
	log.WithField("file", file).Debug("creating snap metadata")
```

File: internal/pipe/snapcraft/snapcraft.go [hunk 4]
```diff
@@ -187,7 +187,7 @@ func create(ctx *context.Context, arch string, binaries []artifact.Artifact) err
 	}
 
 	var snap = filepath.Join(ctx.Config.Dist, folder+".snap")
-	log.WithField("snap", snap).Info("creating")
+	log.WithField("snap", snap)
```
Comment (caarlos0) [inline internal/pipe/snapcraft/snapcraft.go]: here as well:

```suggestion
	log.WithField("snap", snap).Info("creating")
```

File: internal/pipe/archive/archive.go [hunk 5]
```diff
@@ -91,7 +91,6 @@ func create(ctx *context.Context, binaries []artifact.Artifact) error {
 	}
 	defer archiveFile.Close() // nolint: errcheck
 	var log = log.WithField("archive", archivePath)
-	log.Info("creating")
```
Comment (caarlos0) [inline internal/pipe/archive/archive.go]: this one you can add back too...

**Comment by julio-garcia-fc** [comment]
Sorry about that. Please check the new commit.

**Comment by caarlos0** [comment]
Thanks!

**Pull request merged**